service/src/main/java/uk/gov/hmcts/reform/fpl/service/OrganisationService.java

```diff
@@ -32,18 +32,23 @@
     private final RequestData requestData;
 
     public Set<String> findUserIdsInSameOrganisation(String localAuthorityCode) {
+        try {
+            return Set.copyOf(getUsersFromSameOrganisationBasedOnReferenceData(requestData.authorisation()));
+        } catch (FeignException.NotFound | FeignException.Forbidden unregisteredException) {
+            log.warn("User not registered in any org in MO", unregisteredException);
+        } catch (FeignException prdFailureException) {
+            log.error("PRD failed", prdFailureException);
```

Comment:
Message could be a bit better, Maybe 'Request for user in same organisation failed'

Reply:
Oops forgot to change that, will do now